docs/games/0008-Fallout4.md

```diff
@@ -0,0 +1,158 @@
+## General Info
+
+- Name: Fallout 4
+- Release Date: 2015
+- Engine: Creation (Gamebryo)
+
+## Uploaded Files Structure
+
+!!! info "Using popular mods as examples."
+
+Uploaded Fallout 4 mods can appear in the following form:
+
+- Files Targeting Data Subfolder
+    - [Unofficial Fallout 4 Patch](https://www.nexusmods.com/fallout4/mods/4598?tab=files)
+    - Root of archive maps to data.
+    - e.g. `/Unofficial Fallout 4 Patch - Main.ba2` maps to `Data/Unofficial Fallout 4 Patch - Main.ba2`.
+
+- Loose Files Targeting `Data` Subdirectory
+    - [Commonwealth Cuts - KS Hairdos - ApachiiSkyHair](https://www.nexusmods.com/fallout4/mods/11402?tab=files)
+
+- Files Targeting Game Root Directory
+    - [xSE PluginPreloader F4](https://www.nexusmods.com/fallout4/mods/33946?tab=files)
+  
+Mods can ship as 'loose files' or '.esp+.ba2' pairs.
+
+## Additional Considerations for Manager
+
+### DLCList.txt
+
+!!! info "Found in: `%LOCALAPPDATA%\Fallout4` after running the game for the first time."
+
+The `DLCList.txt` file in Fallout 4 lists plugin files with the ".esp" and ".esm" extensions that are loaded after the game's own.
```

Comment:
I believe `plugins.txt` also works. Or at least, the file created by Vortex seems to work fine with both the Steam and GOG versions in my experience.

Not sure if there's any preference or difference between `DLCList.txt` and `plugins.txt`, but the latter should probably get a mention in the docs?